src/guidance/translator.cpp

```diff
@@ -0,0 +1,150 @@
+#include <cstddef>
```

Comment:
I still have to see what I am exposing. I am not saying that this should ever make it outside. It is part of my hack to actually test my guidance and see how it would look like in an application...